core/packages/test/.envrc-example

```diff
@@ -2,30 +2,34 @@ source_up_if_exists
 
 # Overrides for E2E stack
 
+## Substrate config
 # export POLKADOT_VER=                # Relaychain version, default value: v0.9.30
 # export POLKADOT_BIN=                # Relaychain binary, default value: $POLKADOT_DIR/target/release/polkadot
 # export CUMULUS_BIN=                 # Relaychain binary, default value: $CUMULUS_DIR/target/release/polkadot-parachain
 # export BEEFY_START_BLOCK=           # Start block to configure beefy client, default value: 1
 # export RELAYCHAIN_ENDPOINT=         # Relaychain endpoint, default value: ws://localhost:9944
 # export PARACHAIN_RUNTIME=           # Runtime type of parachain should be one of snowbase|snowblink|snowbridge
 
-# Run E2E on Goerli
 
+## Eth config for production
 # export INFURA_PROJECT_ID=           # Your Infura project id
 # export ETH_NETWORK=goerli
 # export ETH_NETWORK_ID=5
 # export ETHERSCAN_API_KEY=           # Your etherscan api key
-
 # export ETH_RPC_ENDPOINT=https://goerli.infura.io/v3
 # export ETH_WS_ENDPOINT=wss://goerli.infura.io/ws/v3
 # export BEACON_HTTP_ENDPOINT=https://lodestar-goerli.chainsafe.io
 # export ACTIVE_SPEC=mainnet # For real network just uncomment and set as mainnet here
-# export ETH_FAST_MODE=false  # For local setup only, set as true will hack beacon client for fast slot time
-
-# export DEPLOYER_ETH_KEY=           # Your private key to deploy contracts
+# export DEPLOYER_ETH_KEY=            # Your private key to deploy contracts
 # export BEEFY_RELAY_ETH_KEY=         # Your Beefy relayer account private key
 # export PARACHAIN_RELAY_ETH_KEY=     # Your Parachain relayer account private key
 # export E2E_TEST_ETH_KEY=            # Your E2E test account private key
+# export ETH_RANDAO_DELAY=96          # 3 epochs=96 slots=20mins
+# export ETH_RANDAO_EXP=128           # 1 epoch more than ETH_RANDAO_DELAY 
```

Comment:
Actually, relayers should only get three blocks worth of chances to commit prevrandao. Less is even better.

```suggestion
# export ETH_RANDAO_EXP=999
```

Reply:
Yes, a shorter time window means more safety. Is `ETH_RANDAO_EXP=3` a bit too short for good relayer to response? I would prefer 6.